lyse/analysis_subprocess.py

```diff
@@ -432,17 +445,12 @@ def reset_figs(self):
 
     os.environ['MPLBACKEND'] = "qt5agg"
 
-    import lyse
-    from lyse import LYSE_DIR
     lyse.spinning_top = True
-    import lyse.figure_manager
+
     lyse.figure_manager.install()
 
-    from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT as NavigationToolbar
+    # Where are pylab features used?  Should this be here?
     import pylab
```

Comment:
I'm not entirely sure why this is here. My best guess is that importing pylab has side-effects that alter the default plot styling. If so, that is something that we could fix much better using a mpl style file. 

Ultimately, I'm on board with getting rid of all `pylab` imports across labscript as it is basically an abandoned interface to matplotlib. But those changes are often technically breaking since users might be using the import incidentally (we do recommend them in the docs in a number of places). Which is to say this should be the subject of another PR.

Reply:
Given this I will not make any of my suggested changes to __init__.py at this time.